build.proj

```diff
@@ -41,11 +24,9 @@
     </Project>
     <!- signing must happen before packaging -!>
     <Project Include="src\sign.builds" />
-    -->
     <Project Include="src\packages.builds" Condition="'$(BuildPackages)'=='true'">
```

Comment:
Yes, that's a problem.  Devs need to build these and run the validation (closure/cycle) that occurs during the build to make sure they don't break CI.

If you want to pull this out of build.cmd do that in a separate commit and make sure we have the right workflow for local build and CI so that we still get this coverage.